docs/pages/repo/docs/reference/command-line-reference/run.mdx

```diff
@@ -269,7 +269,7 @@ Passing `--no-daemon` instructs `turbo` to avoid using or creating the standalon
 
 `type: string`
 
-Set type of output logging. Defaults to "outputMode" for the task in `turbo.json`.
+Set type of output logging. Defaults to [`outputMode`](/repo/docs/reference/configuration#outputmode) for the task in `turbo.json` (if set).
```

Comment:
```suggestion
Set type of output logging. This will override [`outputMode`](/repo/docs/reference/configuration#outputmode) if defined in `turbo.json`.
```